doc/doxyfile.in

```diff
@@ -63,12 +63,18 @@ PROJECT_BRIEF          =
 
 PROJECT_LOGO           =
 
+# With the PROJECT_ICON tag one can specify an icon that is included in the tabs
+# when the HTML document is shown. Doxygen will copy the logo to the output
+# directory.
+
+PROJECT_ICON           =
+
 # The OUTPUT_DIRECTORY tag is used to specify the (relative or absolute) path
 # into which the generated documentation will be written. If a relative path is
 # entered, it will be relative to the location where doxygen was started. If
 # left blank the current directory will be used.
 
-OUTPUT_DIRECTORY       = "@DOXYGEN_OUTPUT_DIR@"
+OUTPUT_DIRECTORY       = @DOXYGEN_OUTPUT_DIR@
```

Comment:
If my commit message was correct https://github.com/SFML/SFML/commit/1b71e3d389eab1babfb2287e1dd9d13415a50463, we need to keep add quotes around cmake variables that could contain spaces.

Reply:
In my experience you never need to put quotes around the expansion of variables that represent paths even if those variables expand into a path that contains a space. For years I've heard this advice and have asked many CMake expert, including the man who invented CMake, and nobody has been able to prove that you need to quote the expansion of such variables.

Reply:
Note this is not a CMake script. We are generating a doxygen configuration file here.

Citing at [doxygen documentation](https://www.doxygen.nl/manual/config.html#config_format):
> If the value should contain one or more blanks it must be surrounded by quotes ("...").

Reply:
Ah, I was wondering about the quotes, but I too build without spaces in path names. Let me test that real quick.

Reply:
Yet they fixed stuff to support it without quotes in 1.9: https://github.com/doxygen/doxygen/commit/ef91bacb7a69bbf7bccb4a864698cc003aabac66

It's a bit contradicting, especially since the upgrade tooling is the one removing the quotes.

I've tested it just fine on Windows.

Reply:
I guess the update tool removes quotes because there are no spaces in CMake variable names before they are substituted. Anyways it looks to work without quotes for me too.

Reply:
> I guess the update tool removes quotes because there are no spaces in CMake variable names before they are substituted.

You're right. If I use quotes with spaces, it keeps the quotes.

I guess, it doesn't hurt to just add them